validator/README.md

```diff
@@ -0,0 +1,32 @@
+# Validator
+
+Validator is used to get the generator configuration (e.g [statsd stress parameters.yml](https://github.com/aws/amazon-cloudwatch-agent-test/blob/2c859b71d067e482985b9c57ca2d2617de8a7795/test/stress/statsd/parameters.yml)) to generate the required loads (e.g 1000 statsd metrics, 1000 logs for a single log g file, etc) to CloudWatchAgent and get the required [validation metrics/ validation logs/ validation traces](https://github.com/aws/amazon-cloudwatch-agent-test/blob/2c859b71d067e482985b9c57ca2d2617de8a7795/test/stress/statsd/parameters.yml#L20-L64) to be validated and validate with an appropriate action (e.g [stress validates](https://github.com/aws/amazon-cloudwatch-agent-test/blob/2c859b71d067e482985b9c57ca2d2617de8a7795/validator/validators/validator.go#L20-L21) the required metrics [within bounds](https://github.com/aws/amazon-cloudwatch-agent-test/blob/2c859b71d067e482985b9c57ca2d2617de8a7795/validator/validators/stress/stress_validator.go#L199-L245))
+
+## Architectural
+![](Architectural.png)
+
+## Validator Type
+
+| Name            | Description                                                                                            |
+|-----------------| -------------------------------------------------------------------------------------------------------|
+|`performance`    | [Record CloudWatchAgent's performance metrics](https://github.com/aws/amazon-cloudwatch-agent-test/tree/main/validator/validators/performance//performance_validator.go) by using procstat (e.g cpu_usage) and send it to DynamoDB                                        |
+|`stress`         | [Record CloudWatchAgent's performance metrics](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/validators/stress/stress_validator.go) when sending high metrics/logs/traces loads and ensure the performance stays consistent between                            |releases                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
```

Comment:
nit: `releases` is being excluded from the table and looks like it should be before `|` anyways.

Reply:
Thanks for the nice catch. Done fixing it !!!